action.go

```diff
@@ -343,7 +343,7 @@ func (a Action) String() (str string, err error) {
 // permission is found, the default one `default` is used.
 // The first permission that is found to apply to an operation, but
 // doesn't allow the operation to run, will fail the verification globally
-func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
+func (a Action) VerifyACL(acl ACL, keyring io.Reader, verifyACLs bool) (err error) {
```

Comment:
I'm wondering if we should use something other than verifyACLs for the flag that is a bit clearer.

Perhaps onlyVerifyPubkey or something like that?